src/Routing/Duplex.purs

```diff
@@ -64,9 +71,12 @@ instance applicativeRouteDuplex :: Applicative (RouteDuplex i) where
 instance profunctorRouteDuplex :: Profunctor RouteDuplex where
   dimap f g (RouteDuplex enc dec) = RouteDuplex (f >>> enc) (g <$> dec)
 
+-- | Use given codec to parse a value of type `o` out of String
+-- | (typically representing part of URL) or produce a `RouteError` if the parsing fails.
```

Comment:
I think we could clarify what "part of a URL" means. The `String` is always expected to be the path, query parameters, and hash.